src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.Helix.targets

```diff
@@ -326,7 +326,18 @@
     </ItemGroup>
 
     <WriteItemsToJson JsonFileName="%(HelixWorkItemList.BuildCompleteJson)" Items="@(BuildComplete)" />
-    <Message Text="Wrote build complete JSON for @(BuildComplete->Count()) Queues." />
+    <Message Text="Wrote job-start (build complete) JSON for @(BuildComplete->Count()) Queues." />
+	
+    <CreateItem Include="%(HelixWorkItemList.BuildCompleteJson)" AdditionalMetadata="RelativeBlobPath=JobStartJsonMessages.json">
+      <Output TaskParameter="Include" ItemName="JobStartJsons" />
+    </CreateItem>
```

Comment:
Or
```
<ItemGroup>
  <JobStartJsons Include="@(HelixWorkItemList->'%(BuildCompleteJson)')">
    <RelativeBlobPath>JobStartJsonMessages.json</RelativeblobPath>
  </JobStartJson>
</ItemGroup>
```
This would do the same as @dagood's suggestion, but preserve any metadata already in `HelixWorkItemList` items.